ironic.conf

```diff
@@ -30,7 +31,7 @@ send_sensor_data_interval = 160
 
 [deploy]
 default_boot_option = local
-erase_devices_metadata_priority = 10
+erase_devices_metadata_priority = 0
```

Comment:
This also is likely unintentional but changes the cleaning knob which addresses an issue discovered in the field on HP Hardware. We'll want to revert this line's change at some point.